src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -586,6 +586,9 @@
                                 "any_gpu": {"type": "boolean", "default": false},
                                 "disable_ibverbs": {"type": "boolean", "default": false},
                                 "less_resources": {"type": "boolean", "default": false}
+{% if version >= "4.2" %}
```

Comment:
As discussed, add a new version (4.3) for this, and update the changelog in the document.